db/operations_test.go

```diff
@@ -0,0 +1,199 @@
+package db_test
+
+import (
+	"github.com/luciodesimone/golang-bootcamp/beer"
+	"github.com/luciodesimone/golang-bootcamp/db"
+	"os"
+	"reflect"
```

Comment:
I see that you may be going far too deep on this exercise :nerd_face:

Reply:
I saw you were using reflect to test whether or not the software had created the structures correctly.

While that may be useful or understandable on interpreted languages, since this is a compiled language, those methods won't ever return something that is different that the struct defined.